src/todolist.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable camelcase */
+export default class TodoList {
+a
+
+constructor() {
+  const savedTasks = JSON.parse(localStorage.getItem('tasks'));
+  if (savedTasks) {
+    this.tasks = savedTasks;
+  } else {
+    this.tasks = [];
+  }
+}
+
+displayTasks() {
+  this.sortAndSave();
+  const check = 'fa-check';
+  const uncheck = 'fa-square-o';
+  // eslint-disable-next-line camelcase
```

Comment:
- [ ] Please remove these eslint disables and fix any issues that appear. The ESlint warnings usually are a sign of badly organized code so it's best to try to fix these issues.